util/utils.go

```diff
@@ -198,3 +205,32 @@ func BeforeVersion(check, cur string) bool {
 func BoldText(s string) string {
 	return fmt.Sprintf("\033[1m%s\033[0m")
 }
+
+func OsExeFileName(s string) string {
+	if runtime.GOOS == "windows" {
+		return s + ".exe"
+	}
+	return s
+}
+
+func ArrayContainsEnvVar(arr []string, ev string) bool {
+	// Function required until Go 1.9
+	ev = strings.ToLower(ev) + "="
+	for i := range arr {
+		if strings.Index(strings.ToLower(arr[i]), ev) == 0 {
+			return true
+		}
+	}
+	return false
+}
+
+func ReplaceEnvVarIfExists(arr []string, ev string, val string) []string {
+	ev_lower := strings.ToLower(ev)
+	for i := len(arr) - 1; i >= 0; i-- {
+		if strings.Index(strings.ToLower(arr[i]), ev_lower) == 0 {
```

Comment:
This should append a `=` to `ev_lower` so that env var like `IPFS_PATH_PROXY` or whatever it might be is not replaced.